reconciletags/reconciletags.py

```diff
@@ -25,8 +25,17 @@
 import os
 import subprocess
 
+from retrying import retry
+
 
 class TagReconciler:
+    def retry_on_calledprocesserror(exception):
+        return isinstance(exception, subprocess.CalledProcessError)
+
+    def retry_on_jsonerror(exception):
+        return isinstance(exception, ValueError)
+
+    @retry(retry_on_exception=retry_on_calledprocesserror)
```

Comment:
let's put a backoff on these retries, and only try a few times before we just give up completely. maybe wait 2-4 seconds between each try, and try something like 4-5 times?